controllers/provisioners/eks/helpers.go

```diff
@@ -489,6 +489,13 @@ func (ctx *EksInstanceGroupContext) GetLabelList() []string {
 	return labelList
 }
 
+func min(a, b int64) int64 {
```

Comment:
Probably move to common/utils.go - or you can also use math.Min() but it takes float. Whatever makes more sense 👍